wbooks/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import logo from './wbooks_logo.svg';
+import notifications from './notifications.svg';
+import addBook from './add_book.svg';
+import search from './search.svg';
+import bookIcon from './default_book.svg'
+import './App.css';
+import Books from './Books'
+
+class App extends Component {
+
+  constructor() {
+    super();
+    this.state = {value: ''};
+    this.handleFilter = this.handleFilter.bind(this);
+  }
+
+  handleFilter(event) {
```

Comment:
You can use arrow functions to avoid the `.bind`
Just define the method like this:
```js
handleFilter = (event) => {
```
Then remove the bind in the constructor.